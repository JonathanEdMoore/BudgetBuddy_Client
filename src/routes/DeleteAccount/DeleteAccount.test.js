import React from 'react'
import ReactDOM from 'react-dom'
import DeleteAccount from './DeleteAccount'
import { BrowserRouter } from 'react-router-dom'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

describe('DeleteAccount smoke test', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<BrowserRouter><DeleteAccount /></BrowserRouter>, div)
    ReactDOM.unmountComponentAtNode(div)
  })
})

describe('DeleteAccount snapshot test', () => {
  const wrapper = shallow(<DeleteAccount />)
  expect(toJson(wrapper)).toMatchSnapshot()
})