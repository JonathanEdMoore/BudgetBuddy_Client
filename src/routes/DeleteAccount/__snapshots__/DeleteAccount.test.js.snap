// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<div>
  <main
    role="main"
  >
    <header>
      <MainHeader />
      <h2>
        Delete Account
      </h2>
    </header>
    <section>
      <LoginForm
        onDeleteSuccess={[Function]}
      />
    </section>
  </main>
</div>
`;
